docs/_includes/notification.html

```diff
@@ -19,16 +19,9 @@
             rgba(50, 49, 93, 0.25) 0px 12px 25px -11px,
             rgba(0, 0, 0, 0.3) 0px 7px 15px -10px,
             rgba(10, 37, 64, 0.35) 0px -2px 6px 0px inset;">
-    <p>
-        Our first official <strong><a href="/docs/contributor-sync/" style="text-decoration: underline; color: #0946EC"
-    >VSecM Contributor Sync</a></strong> will be on…
- <br><br>
-        <strong>Friday, 2023-09-29</strong> (<em>instead of Thursday</em>)
-        at <strong>8:00am</strong> Pacific time.
-<br><br>
-        Anyone is welcome to join.
-<br><br>
-        <a href="/docs/contributor-sync/" style="text-decoration: underline; color: #0946EC"
-        >Check out the details about how to join the meeting</a>.
-    </p>
+    <p>The next <strong><a href="/docs/contributor-sync/"
+    style="text-decoration: underline; color: #0946EC"
+    >VSecM Contributor Sync</a></strong> will be on…<br>
+    <strong>Thursday, 2023-10-26</strong>
+    at <strong>8:00am</strong> Pacific time.</p>
```

Comment:
Filed an issue to dynamically generate these (from a source of truth) instead of creating a PR whenever something new is announced.

It’s not urgent, though; we can live this way for a while.